plugins/dicom_viewer/plugin_tests/data/000000.dcm.sha512

```diff
@@ -0,0 +1 @@
+9a3ee0830fd1b4da2babe5cdb30d4fa0021898c8ba3167b08bffb719e89894087f3cf27d2307cec440c71f13589e39b8fa2c5d094ce90ae5240c522870a52068
```

Comment:
Can you add a newline to the end of each of these 4 files? The old `hashsum_download` logic was not doing so.